main.cpp

```diff
@@ -60,12 +52,12 @@ int main(void)
     GREENTEA_SETUP(90, "default_auto");
 #endif
 
-    // Disable deep sleep
-    sleep_manager_lock_deep_sleep();
-
     tfm_log_printf("Starting TF-M PSA API tests\r\n");
 
-    return val_entry();
+    uint32_t retval = val_entry();
+    TEST_ASSERT_EQUAL_UINT32(0, retval);
```

Comment:
Moving to TEST_ASSERT here on the return value is a change that deserves its own commit and justification. It's a good change, just doesn't fit the `TF-M IRQ API now removed for the IPC model` commit's purpose.